src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promiseOne = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promiseSecond = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+promiseOne.then((success) => {
+  const div = document.createElement('div');
+
+  div.classList.add('message');
+  div.textContent = success;
+  document.body.appendChild(div);
+});
+
+promiseSecond.catch((error) => {
+  const div = document.createElement('div');
+
+  div.classList.add('message', 'error-message');
+  div.textContent = error;
```

Comment:
Currently, `div.textContent = error;` will set the text content to the string representation of the error object. To display the error message, use `div.textContent = error.message;` instead.